packages/grafana-ui/src/components/CustomScrollbar/CustomScrollbar.tsx

```diff
@@ -26,6 +26,7 @@ interface Props {
   autoHeightMin?: number | string;
   updateAfterMountMs?: number;
   onScroll?: React.UIEventHandler;
+  divId?: string;
```

Comment:
Does it finally work with `id` naming?

Reply:
No it doesn't, I think it clashes with the existing HTML attribute `id`.